### PR TITLE
Prepare/release 1.6.3

### LINKED ISSUE #827: Release 1.6.3
These instructions assume you have `$VERSION`, `$PROJECT`, and `$REPO` environment variables set in your shell (e.g. `1.4`, `pgautofailover`, and `pg_auto_failover`). With those set, code from most steps can be copy-pasted.

# Prepare Project

  - [x] Ensure all needed changes are in the relevant `release-x.y` branch. `git log --cherry-pick --no-merges release-x.y...master` can be helpful. Be sure to cherry-pick changes in the same order they were _merged_ to the main branch (but do not cherry-pick merge commits themselves)
  - [x] Add a `CHANGELOG` entry in the ~~`master`~~ `release-x.y` branch summarizing meaningful changes ( will merge release branch into master after releasing to update the changelog there )
  - [x] Update vendored in libraries under `src/bin/lib`.  Verify `README.md` has a reference to each library in the folder.
    - [x] PG : Compare the contents of  `snprintf.h , snprintf.c` and `strerror.c` in folder `src/bin/lib/pg` against PG equivalents in recent PG sources under `src/port` and apply changes to respective places.
    - [x]  Parson : Compare the contents of `src/bin/lib/parson` vs the source  from https://github.com/kgabis/parson
    - [x] ini.h: Compare `src/bin/lib/libs/ini.h` vs source at https://github.com/mattiasgustavsson/libs. Apply changes.
    - [x] log.c: Compare contents of `src/bin/lib/log` vs its source from https://github.com/rxi/log.c . Apply changes as necessary
  - [x] Update version number in
  	- [x] `src/bin/pg_autoctl/defaults.h`
  	- [x] `docs/conf.py`
  	- [x] `Makefile`
  	- [x] `src/bin/pg_autoctl/azure.c`
  - [x] Get _complete_ approval for the commit you're tagging before creating or pushing any tags. Tags should be immutable, so do **not** proceed to the next step until you're sure you have _everything_ you want in your release branch
  - [ ] Use `git tag -a -s v$VERSION` to create an annotated, signed tag for the release. Summarize the release in the one-line tag annotation (beneath 52 characters). Push the tag with `git push origin v$VERSION`
  - [ ] Visit the project's releases page (e.g. `open https://github.com/citusdata/$REPO/releases`)
    - [ ] Create a new release object for your git tag (i.e. `v$VERSION`). Leave the description blank (it will auto-fill with the tag description)

As for signing tags, use your own key, and ensure it's [known to GitHub](https://help.github.com/articles/adding-a-new-gpg-key-to-your-github-account/).

# Update OS Packages
  - [ ] Make sure you have the latest tooling installed from the https://github.com/citusdata/tools repo, using `(sudo) make install`
  - [ ] Go to your locally checked out version of the https://github.com/citusdata/packaging repo.
  - [ ] Create a packaging PR using `update_os_package.pl all pgautofailover $VERSION`
  - [ ] Test the release build locally: `citus_package -p=debian/buster  -p=debian/stretch -p=ubuntu/bionic -p=ubuntu/xenial -p ubuntu/focal -p=el/8   -p=el/7  -p=ol/7 local release 2>&1 | tee citus_package.log`
  - [ ] Confirm for some of the packages that the contents look decent using `dpkg -c package.deb` and `rpm -qlp package.rpm`
  - [ ] Ensure no warnings or errors are present: `grep -Ei '(warning|\bi|\be|\bw):' citus_package.log | sort | uniq -c | grep -v "\.c"`. Ignore any errors about `--disable-dependency-tracking`
  - [ ] Get changes reviewed;
  - [ ] Confirm that CI is happy
  - [ ] Use the "squash" strategy to close the PR
  - [ ] Ensure both Travis builds completed successfully (new releases should be in packagecloud, you should expect 52 items when searching for new version)

### PR DESCRIPTION
**Pull request by DimCitus**
Fixes #827

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+* New ncurses dashboard with command pg_autoctl watch (#809)
+
+#### Changed
+* Allow setting maximum-backup-rate on create postgres step (#812)
+
+#### Fixed
+* Work around pg_replication_slot_advance xmin maintenance bug. (#815)
+* Fix "Current State" to "Reported State", and a docs cross-ref.
+* Monitor config set postgresql.pg_ctl bug fix (#818)
+* Refrain from using atexit() to clean-up our logs semaphore. (#811)
```
Comment (JelteF) [inline CHANGELOG.md:20]: Is this actually true? I believe that in the latest version of the PR we are still using `atexit`, we just use it in a different way.

**Review by JelteF — APPROVED** [review]
I left one comment, other than that this looks fine.

File: CHANGELOG.md (hunk 1 above)
Comment (DimCitus) [reply CHANGELOG.md:20]: Ah yeah I could rephrase, I just copied the PR's title desc.

Comment (DimCitus) [reply CHANGELOG.md]: Done now.

**Review by JelteF — APPROVED** [review]

**Pull request merged**